hazelcast-jet-core/src/main/java/com/hazelcast/jet/config/JobConfig.java

```diff
@@ -65,6 +67,7 @@
     private boolean storeMetricsAfterJobCompletion;
 
     private Map<String, ResourceConfig> resourceConfigs = new LinkedHashMap<>();
+    private Map<String, String> serializerConfigs = new LinkedHashMap<>();
```

Comment:
Do we need to preserve the iteration order? `HashMap` is cheaper. For `resourceConfigs` it's probably important because two added JARs can have duplicate classes, and the first one should be used, but here we don't allow duplicates.